protozoo/tiktorch_config_keys.py

```diff
@@ -37,24 +37,17 @@ def __init__(self, loss_class: Optional[Type] = None, **loss_kwargs: Any):
         self.loss_kwargs = loss_kwargs
 
 
+@dataclass
 class Callback:
     event: Events
     function: Callable[[Engine], None]
```

Comment:
I don't quite understand the signature of the `Callable` here. Could you explain this a bit?

Reply:
typing.Callable:
Callable[[arg1, arg2], ret1, ret2]

Reply:
What confused me more was the return-value (or rather the absence of a return value).
But this makes sense, Callbacks only change internal states of the engine.
Would not hurt to add some documentation on this though ;).

Reply:
For completness sake, I have to correct my above comment.
The typing.Callable signature is:
`Callable[[Arg1Type, Arg2Type], ReturnType]`
thus the above example should have been:
`Callable[[Arg1Type, Arg2Type], Tuple[ReturnType1, ReturnType2]]`